t05_rule_with_trailing_slash.py

```
@@ -0,0 +1,6 @@
+from base import app
+
+
+@app.route("/about/")
+def hello():
+    return f"Hello, X!"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)